src/Microsoft.AspNet.Razor/CodeGenerators/Visitors/CSharpUsingVisitor.cs

```diff
@@ -11,7 +11,11 @@ namespace Microsoft.AspNet.Razor.CodeGenerators.Visitors
 {
     public class CSharpUsingVisitor : CodeVisitor<CSharpCodeWriter>
     {
-        private const string TagHelpersRuntimeNamespace = "Microsoft.AspNet.Razor.Runtime.TagHelpers";
+        private static readonly string[] TagHelpersRuntimeNamespaces = new[]
+        {
+            "Microsoft.AspNet.Razor.TagHelpers",
+            "Microsoft.AspNet.Razor.Runtime.TagHelpers.Internal"
```

Comment:
It's identical to what we had before. We need the `Internal` namespace to enable access to the variety of `TagHelper` enabling types such as `TagHelperExecutionContext`, `TagHelperRunner` etc. We could at some point remove these namespaces and generate all the required fields using their fully qualified names.